### ISSUE TITLE
URGENT - Footer changes

### ISSUE THREAD

**Issue by bobsummerwill** [ETCGrantsDAO/etc-grants-dao-website#176]
For the Footer, if you look at this preview, you will see an older version where ETC Cooperative was on its own on the left and then there were three columns for Bitmain, Antpool and ETC:
https://deploy-preview-157--elegant-blini-3fea6b.netlify.app/

So we would have it something like that, except with EGD on the left, and then "Sponsors" for Coop, Bitmain and Antpool on the right.    We would delete the "Ethereum Classic" ones.

Also, the Footer is really screwed up looking on mobile.   When the recolumning here is done for Desktop, that will be a second pass.

The links for the new ETC Grants DAO column on the left will be the ones from the Resources section.
We are still waiting on a logo for that, so maybe just use the Ethereum Classic one for now?

**Comment by gd8097019** [comment]
Hi I commited for footer in new PR but it goes directly to here: [URGENT - Footer changes #176
](https://github.com/ETCGrantsDAO/etc-grants-dao-website/pull/178)

<img width="1278" alt="Screen Shot 2023-01-26 at 14 17 01" src="https://user-images.githubusercontent.com/102132498/214823157-17f6119c-1c82-4ed1-b866-b078bb052c62.png">

**Comment by gd8097019** [comment]
By the way I can add only twitter and telegram link for "ETC Grants DAO". If you provide `github` and `wechat` icons I can added to below.

**Comment by bobsummerwill** [comment]
Thanks!
